src/panels/developer-tools/ha-panel-developer-tools.ts

```diff
@@ -67,6 +67,9 @@ class PanelDeveloperTools extends LitElement {
                 "ui.panel.developer-tools.tabs.statistics.title"
               )}
             </paper-tab>
+            <paper-tab page-name="yaml">
```

Comment:
Should it be the first ?

Reply:
:shrug:

I feel like if we change the first tab people riot. If we don't put it first people riot. Do we just wait and see?